inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -205,6 +205,18 @@ public function execute( HttpQueryInterface $query, array $input_variables ): ar
 			}
 		}
 
+		// If the query is a search query and the search term is empty, return a
+		// collection with no results. This is to exit early and avoid making an
+		// unnecessary API call.
+		if ( 'search' === $key && empty( $input_variables['search'] ) ) {
```

Comment:
Is there a built in way to do this already that I might have missed? I essentially want to skip the initial search call if the provided search term is empty. This happens when a block is first created and I don't think it should occur. Until a search term is provided, this should be skipped.